### PR TITLE
Bug 1156727 - Build up a connection between "system" and "system-remote"

### PR DESCRIPTION
**Pull request by luke-chang**

### CODE REVIEW

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=13b2c0e959627f82dc02033e1a2021489b5a34cd

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=ef521336f3ceafd977b3b7e19d4aef346beee86e

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=bb48c844054e6d4b2c9773e13e69c0aae9a6ac21

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=88e61378120b5fdd3e0dfcd617a93efaf171f637

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=e4d12d031585374d0b50d7658f1ce52fa2274d59

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=62216326c89c51bd33919746be9700c7441196ee

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=53403e1e79b86bab3beb1be0d5661f02b6f62289

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=90d1896e97802eada0103acf4fa078575b7fd451

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=5cbce9f48b51a7ab391277b9944795a519a4ab2c

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=6c5ff5607dfc8b2d68f5294afe5552d8ce397e6d

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=0246845819f313bf5d6a3c134c67e0d103190def

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=30c3ac9136e115772e9f55c10dfdf3a7b7a2b231

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=c1fd85dedab39756faee47369dcfa67f57c8a45e

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=cfd22306333d6f316f8c63891c69819698f1e475

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=e94943a7431faa45b8bf987e4cf27ed885ed9535

File: apps/system/js/wallpaper_manager.js [hunk 1]
```diff
@@ -228,7 +228,8 @@
 
       // How big (in device pixels) is the screen in its default orientation?
       var screenWidth, screenHeight;
-      if (OrientationManager && !OrientationManager.isDefaultPortrait()) {
+      if (window.OrientationManager &&
```
Comment (alivedise) [inline apps/system/js/wallpaper_manager.js]: This was removed by bug 1094759

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=ebc6e1ebea643a7e0e6c84741bdf976cb7186ee6

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=b7fc2817e40c5c98521b74e6987d7054566b7ee2

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=a6182dac873a7ee60f3e6a5382334a6dcccda940

**Pull request merged**